starter-code/pizza.js

```diff
@@ -1 +1,52 @@
 // Write your Pizza Builder JavaScript in this file.
+
+// ITERATION 1
+
+var $buttonPepperonni = $('.btn-pepperonni');
+var $buttonMushroom = $('.btn-mushrooms');
+var $buttongreenPeppers = $('.btn-green-peppers');
+
+
+$buttonPepperonni.click (function () {
+    var $pepperonni = $('.pep');
+    $pepperonni.toggleClass('hide');
+    $(this).toggleClass('active');
+    updatePriceItem(1);
+});
+
+$buttonMushroom.click (function () {
+    var $mushroom = $('.mushroom');
+    $mushroom.toggleClass('hide');
+    $(this).toggleClass('active');
+});
+
+$buttongreenPeppers.click (function () {
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.